server/reload_test.go

```diff
@@ -34,7 +34,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/nats-io/jwt/v2"
+	"github.com/nats-io/jwt"
```

Comment:
Why remove v2? I think there was conscious effort to upgrade to v2 for the future.

Reply:
Mistake from auto goimports... will fix

Reply:
This was fixed.